lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -73,6 +73,9 @@ def model_create_prm_any_baseline_building(model, building_type, climate_zone, h
         space_type_apply_internal_loads(space_type, set_people, set_lights, set_electric_equipment, set_gas_equipment, set_ventilation, set_infiltration)
       end
     end
+    if /prm/i =~ template
```

Comment:
Do you think you could add some comments here to explain what's happening there?

Reply:
Modified.